schema/features/element-hiding.ts

```diff
@@ -0,0 +1,31 @@
+import { Feature } from '../feature';
+
+export type ElementHidingFeature<VersionType> = Feature<
+    {
+        useStrictHideStyleTag?: boolean;
+        rules: Array<{
+            selector?: string;
+            type: 'hide-empty' | 'hide' | 'closest-empty' | 'override' | 'modify-style' | 'modify-attr' | 'disable-default';
+            values?: Array<{
+                property: string;
+                value: string;
+            }>;
+        }>;
+        domains: Array<{
+            domain: string | string[];
+            rules: Array<{
+                selector?: string;
+                type: 'hide-empty' | 'hide' | 'closest-empty' | 'override' | 'modify-style' | 'modify-attr' | 'disable-default';
+                values?: Array<{
+                    property: string;
+                    value: string;
+                }>;
+            }>;
+        }>;
+        unhideTimeouts?: number[];
+        mediaAndFormSelectors?: string;
+        adLabelStrings?: string[];
+        [key: string]: any;
```

Comment:
Is this a typo? Maybe meant to be `hideTimeouts`?

```suggestion
        hideTimeouts?: number[];
```

Reply:
What does removing this  `[key: string]: any;` do?
Do tests fail?

Again I would fast follow with trying to remove this, but essentially it's currently an open contract for more keys.